EasyPost.EasyVCR/Mode.cs

```diff
@@ -0,0 +1,25 @@
+namespace EasyPost.EasyVCR
+{
+    /// <summary>
+    ///     Modes of operation for the VCR system.
+    /// </summary>
+    public enum Mode
+    {
+        /// <summary>
+        ///     No recording or playback. Make HTTP requests as normal.
+        /// </summary>
+        Bypass,
```

Comment:
Just a note that `Bypass` doesn't really appear in the comment headers of other functions when the other 3 modes do. May be good to add it in those places?